### ISSUE TITLE
When multiple background images sources marked with a star are available none is selected by default

### ISSUE THREAD

**Issue by ttomasz** [openstreetmap/iD#8382]
In Poland there are two WMS services providing orthophotomap one for high resolution images and one for standard resolution images. Both are marked with a star as preferred/default basemaps for this region (see screenshot). 

By default iD doesn't select any of them and user is greeted with a black background which can be confusing.

I think a better option would be to select one at random or add some kind of priority ranking for WMS addresses so iD could resolve which imagery it should load by default.

![image](https://user-images.githubusercontent.com/31594210/110237493-b23c6580-7f3c-11eb-817d-1fd9604ac179.png)

Location of screenshot: https://www.openstreetmap.org/edit#map=18/52.20654/20.86821
iD version: 2.19.6

**Comment by manfredbrandl** [comment]
There is indeed 2.19.6 on openstreetmap.org since 3/3 but CHANGELOG shows 2.19.5 as latest release.

**Comment by Asteliks** [comment]
The problem seems to be that ID chooses the alphabetically first layer that has the “"best": true” for the region. In this case the alphabetically first layer is the overlay called “Geoportal 2 Nazwy ulic” thus the layers are not chosen, and the background is black.

**Comment by maro-21** [comment]
I think the issue no longer occurs. Am I right? Can we close it?

**Comment by tyrasd** [comment]
yes, this was fixed a while ago. First with https://github.com/osmlab/editor-layer-index/pull/1274, and then eventually more generic with https://github.com/openstreetmap/iD/commit/e03d3b48fa371e15c3d85e105375200862751446#diff-f1fe176dad1bfb3b10c97d5e1dc510d1aee1272e9461d0e011e9e16216fb02eeR470: iD will not consider overlays with `best` flag.
